src/pages/install/{installConfig.agentName}.js

```diff
@@ -194,6 +196,14 @@ const InstallPage = ({ data, location }) => {
 
   const headings = walkthroughSteps.map(({ stepHeadings }) => stepHeadings);
 
+  useEffect(() => {
```

Comment:
could you leave a comment here or add something to the commit message for `cae1f96` to give more context for this change? just to make sure if someone came back to this in the future if we could easily understand why this change was made